src/apps/lwaftr/lwaftr.lua

```diff
@@ -299,21 +290,21 @@ end
 
 local function decrement_ttl(pkt)
    local ipv4_header = get_ethernet_payload(pkt)
-   local checksum = bnot(ntohs(rd16(ipv4_header + o_ipv4_checksum)))
+   local chksum = bnot(ntohs(rd16(ipv4_header + o_ipv4_checksum)))
```

Comment:
As above, the intention is to disambiguate the local variable from the module-level library reference. This time I changed the local variable name even if it appears 10 times, while the library reference is only used once.